generators/exercises/yacht.py

```diff
@@ -0,0 +1,33 @@
+FUNC_PROTO = """\
+#include "vendor/unity.h"
+
+#include <stdint.h>
+
+typedef enum {
+    CHOICE = 0,
+    ONES,
+    TWOS,
+    THREES,
+    FOURS,
+    FIVES,
+    SIXES,
+    LITTLE_STRAIGHT,
+    BIG_STRAIGHT,
+    FULL_HOUSE,
+    FOUR_OF_A_KIND,
+    YACHT,
+} category_t;
+
+extern int score(category_t category, const uint16_t *dice);
+"""
+
+def array_literal(digits):
+    return str(digits).replace('[', '{').replace(']', '}')
```

Comment:
I would find this cleaner:
```suggestion
    return '{' + ', '.join(str(d) for d in digits) + '}'
```